docs/ui/vscode.rst

```diff
@@ -0,0 +1,155 @@
+Editing lambdapi source code with `VSCode`_
+===========================================
+
+There is an extension for VSCode >= 1.37 derived from VSCoq. To install
+it do from the ``lambdapi`` repository:
+
+.. code:: bash
+
+   cd editors/vscode/
+   make clean
+   make
+
+This requires to have ``npm`` and ``node-typescript`` installed:
+
+.. code:: bash
+
+   sudo apt install npm node-typescript
+
+*note*: as of today the vscode mode requires a full lambdapi install, it
+won’t unfortunately run from a developer tree.
+
+Overview
+--------
+
+This extension provides syntax highlighting, unicode characters easy
+insertion and proof navigation as in CoqIde or ProofGeneral.
+
+It is still under development, thus might be incomplete and buggy.
+
+Features
+--------
+
+Proof navigation
+^^^^^^^^^^^^^^^^
+
+Goals are visualised in a panel on the right side of the editor.
+
+* You can navigate the proof with `keybindings <#keybindings>`__.
+* “Cursor mode” can be toggled to navigate the proof with the cursor.
+
+Snippets
+^^^^^^^^
+
+Type the advised snippets described below, then press Enter or Tab to
+confirm adding the chosen Unicode character.
+
+--------------
+
+Greek letters
+^^^^^^^^^^^^^
+
+Typing ```letter`` will suggest a corresponding unicode greek letter (for
+instance ```b`` will suggest β). Some greek letters are present in a “var” form
+as in LaTeX, accessible with ```vletter`` (for instance, ```f`` will suggest ϕ
+and ```vf`` will suggest φ).
+
+--------------
+
+Fonts
+^^^^^
+
+* ```dletter`` (for instance \`dn): double-struck letter (ℕ)
+
```

Comment:
do we need an empty line between each item? idem later

Reply:
perhaps we should write `<letter>`  instead?

Reply:
We only need to have a blank line before the items. It's been fixed.